Source/Core/InputCommon/ControllerInterface/GameController/GameControllerDevice.mm

```diff
@@ -0,0 +1,109 @@
+// Copyright 2013 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "InputCommon/ControllerInterface/GameController/GameControllerDevice.h"
+
+#include "Common/Logging/Log.h"
+#include "Common/MathUtil.h"
+
+namespace ciface::GameController
+{
+using MathUtil::GRAVITY_ACCELERATION; // 9.80665
```

Comment:
Maybe move this into the constructor, given it's only used in there.

Reply:
Not like GRAVITY_ACCELERATION is likely to change, but why copy its value in the comment?